### PR TITLE
Better format for cross-platform labelling and training (storing triples) 

### PR DESCRIPTION
**Pull request by jeylau**
Image paths were stored with OS-specific path separators, making labeling/training hard across platforms (e.g., forcing users to create the training data with the machine they train their models on, and occasionally to convert their data). Paths are now stored without separators as MultiIndex (labeled-data, subfolder, imagename), making indexing/merging DataFrames easy and robust; only upon reading an image are the paths reconstructed.
I tested the GUIs as well ☺️

### CODE REVIEW

File: deeplabcut/generate_training_dataset/multiple_individuals_trainingsetmanipulation.py [hunk 1]
```diff
@@ -97,7 +97,6 @@ def create_multianimaltraining_dataset(
     config,
     num_shuffles=1,
     Shuffles=None,
-    windows2linux=False,
```
Comment (MMathisLab) [inline deeplabcut/generate_training_dataset/multiple_individuals_trainingsetmanipulation.py]: maybe we should leave, but keep false / make a depreciation warning, as I imagine some people pass this in their code quite a bit?

**Review by MMathisLab — CHANGES_REQUESTED** [review]
maybe just leave term in and add deprecation warning, i..e, it becomes a dummy variable, but then people won't get the error invalid input , etc etc (i..e, in our colab notebooks, etc)

File: deeplabcut/generate_training_dataset/multiple_individuals_trainingsetmanipulation.py (hunk 1 above)
Comment (jeylau) [reply deeplabcut/generate_training_dataset/multiple_individuals_trainingsetmanipulation.py]: Great point, I'll add the warning!

**Review by AlexEMG — CHANGES_REQUESTED** [review]
- let's add some tests 
- update the examples

**Review by AlexEMG — APPROVED** [review]

**Pull request merged**